zerver/lib/test_classes.py

```diff
@@ -65,6 +65,10 @@
 
 API_KEYS = {} # type: Dict[text_type, text_type]
 
+skip_py3 = unittest.skipIf(six.PY3, "Expected failure on Python 3")
+
+
+
```

Comment:
I think you added this change accidentally when resolving merge conflicts (we had removed that in `master` recently)